modules/ROOT/pages/index.adoc

```diff
@@ -1,88 +1,132 @@
+--
+[discrete]
+=== Design
 
-[%header%autowidth.spread]
-|===
-| Mule 3    | Mule 4
-| xref:3.9@mule-runtime::anypoint-connectors.adoc[Connectors for Mule 3] | xref:connectors::index.adoc[Connectors for Mule 4]
-| xref:3.9@mule-runtime::dataweave.adoc[DataWeave 1 for Mule 3]  | xref:mule-runtime::dataweave.adoc[DataWeave 2 for Mule 4]
-| xref:3.x@apikit::index.adoc[APIkit 3] | xref:4.x@apikit::overview-4.adoc[APIkit 4]
-| xref:3.9@connector-devkit::index.adoc[Connector DevKit] | xref:1.1@mule-sdk::index.adoc[Mule SDK]
-|===
+Create and publish new API specifications from scratch.
+
+* xref:design-center::design-create-publish-api-specs.adoc[Create a reusable API specification]
+* xref:access-management::index.adoc[Plan for access]
+* xref:general::security.adoc[Plan for security]
+--
+
+image::develop-and-test.png[]
+
+--
+[discrete]
+=== Develop & Test
+
+Build and test APIs and integrations.
 
-== Testing Integrations
+* xref:studio::index.adoc[Develop your API]
+* xref:connectors::index.adoc[Connect your data, systems, and apps]
+* xref:mule-runtime::mule-app-dev.adoc[Develop the Mule app that integrates systems, services, APIs, and devices]
+* xref:dataweave::index.adoc[Access and transform data within the Mule app]
+* xref:munit::index.adoc[Build automated tests for your APIs and integrations]
+--
 
-It is best practice to test the integrations you create before you deploy them:
+image::deploy.png[]
 
-* xref:2.1@munit::index.adoc[Testing with MUnit]
-* xref:runtime-manager::troubleshooting.adoc[Troubleshooting]
+--
+[discrete]
+=== Deploy
 
-== Deploying Integrations
+Deploy APIs and apps to your hosting targets, and secure them with policies.
 
-After you test your integration, deploy it to a production environment:
+* xref:runtime-manager::deployment-strategies.adoc[Deployment options]
+* xref:api-manager::getting-started-proxy.adoc[Configure a proxy for your API]
+* xref:api-manager::policies-landing-page.adoc[Create policies to secure your API]
+--
 
-* xref:runtime-manager::deployment-strategies.adoc[Choosing a Deployment Strategy]
-* xref:3.9@mule-runtime::installing.adoc#mule-installation-procedure[Installing Standalone Mule 3]
-* xref:mule-runtime::runtime-installation-task.adoc[Installing Standalone Mule 4]
+image::monitor.png[]
 
-== Managing Integrations
+--
+[discrete]
+=== Monitor
 
-After your integration is running in production, monitor its performance:
+Monitor your APIs and integrations using dashboards, metrics, and visualization.
 
-* xref:monitoring::index.adoc[Anypoint Monitoring]
-* xref:api-functional-monitoring::index.adoc[API Functional Monitoring]
-* xref:runtime-manager::monitoring.adoc[Runtime Monitoring]
+* xref:monitoring::index.adoc[Monitor Anypoint Platform performance from a single user interface]
+* xref:api-functional-monitoring::index.adoc[Monitor your deployed API performance]
+* xref:runtime-manager::monitoring.adoc[Monitor your deployed apps]
```

Comment:
```suggestion
* xref:monitoring::index.adoc[Monitor Anypoint Platform performance from a single user interface]
* xref:api-functional-monitoring::index.adoc[Check your deployed API performance]
* xref:runtime-manager::monitoring.adoc[Supervise your deployed apps]
```

Reply:
@hannanelson do you agree with this rewording?

Reply:
This implies monitoring Anypoint Platform performance, not performance of integrations/applications:
`Monitor Anypoint Platform performance from a single user interface`

Reply:
View metrics for integration apps?